examples/person-detection/person_detection.py

```diff
@@ -23,12 +24,35 @@
 !mkdir -p binaries/person_detection && cd binaries/person_detection && wget https://github.com/antmicro/tensorflow-arduino-examples-binaries/raw/master/person_detection/person_detection.ino.elf # fetch prebuilt binaries
 
 # %% [markdown]
-"""## Take a photo"""
+"""
+## Take a photo
+To change option rerun the cell
+# """
 
 # %%
+import ipywidgets as w
 from IPython.display import Image
-photo = take_photo()
-display(Image(photo))
+
+def handle_selection(obj):
+  if obj['new'] == 'Camera':
```

Comment:
this option failed for me after I first selected the `default` option.

Can we display the default/uploaded file?